Documentation/gettingstarted/servicemesh/envoy-custom-listener.rst

```diff
@@ -0,0 +1,70 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _gs_envoy_custom_listener:
+
+*******************
+L7 Path Translation
+*******************
+
+This example replicates the Prometheus metrics listener which is
+already available via Cilium Agent ``--proxy-prometheus-port`` command
+line option. So the point of this example is not to add new
+functionality, but to show how a feature that previously required
+Cilium Agent code changes can be implemented with the new Cilium Envoy
+Config CRD.
+
+Apply Example CRD
+=================
+
+This example adds a new Envoy listener ``envoy-prometheus-metrics-listener``
+on the standards Prometheus port (e.g. ``9090``) to each Cilium node, translating
+the default Prometheus metrics path ``/metrics`` to Envoy's Prometheus metrics path
+``/stats/prometheus``.
+
+Apply this Cilium Envoy Config CRD:
+
+.. parsed-literal::
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/servicemesh/envoy/envoy-prometheus-metrics-listener.yaml
+
+This version of the CEC CRD is Cluster-scoped, (i.e., not namespaced),
+so the name needs to be unique in the cluster, unless you want to
+replace a CRD with a new one.
+
+.. Note::
+
+    Note that these Envoy resources are not validated by k8s at all, so
+    any errors in the Envoy resources will only be seen by the Cilium
+    Agent observing these CRDs. This means that ``kubectl apply`` will
+    report success, while parsing and/or installing the resources to the
+    node-local Envoy instance may have failed. Currently the only way of
+    verifying this is by observing Cilium Agent logs for errors and
+    warnings.
+
```

Comment:
Also, what happens if the user configures multiple CEC? What happens if they configure Ingress and their own CEC? Are these supported? If not, these should also be called out in the limitations like the validation concern above.